src/components/IPFSSubpathRedirect.tsx

```diff
@@ -0,0 +1,34 @@
+import { CID } from 'multiformats'
+import { useEffect } from 'react'
+
+export const IPFSSubpathRedirect = () => {
+  useEffect(() => {
+    const hashFromBase = extractIPFSHashFromBasePath()
+
+    if (!hashFromBase) {
+      console.info('Base ref does not contain an IPFS hash')
+      return
+    }
```

Comment:
This feels like unnecessary noise since 99% of the time it shouldn't have content hash in the path.
```suggestion
    if (!hashFromBase) return
```